avm/res/data-factory/factory/main.json

```diff
@@ -522,21 +651,21 @@
       }
     },
     "managedPrivateEndpoints": {
-      "type": "array",
+      "$ref": "#/definitions/managedPrivateEndpointsType",
```

Comment:
help me to understand this line of code please, never seen this before

Reply:
Hey @clintgrove, this is a reference to the 'new' `definitions: {}` section that was introduced to ARM templates following the introduction of User-defined types. What it is saying is: _The structure/definition of this parameter can be found in the `definition` map with the key `managedPrivateEndpointsType`:

https://github.com/Azure/bicep-registry-modules/blob/cc97c1df6320387193731fd13db3b7619879aa86/avm/res/data-factory/factory/main.json#L509-L544